### PR TITLE
updated getOrderDetails to use fragment

### PR DESCRIPTION
**Pull request by DenisMela**
The getOrderDetails query should return the order object, not just the _id of the object

### CODE REVIEW

File: src/app/controllers/OrderController.ts [hunk 1]
```diff
@@ -365,9 +435,12 @@ export class OrderController {
       let mutationString = `
                 query getOrderDetails ($orderId: String!){
                     getOrderDetails(order_id: $orderId) {
-                        _id
+                      ...OrderFragment
                     }
                 }
+                `
```
Comment (ralph-dev) [inline src/app/controllers/OrderController.ts:441]: Don't you already have these isnt it a duplicate tick?

**Review by ralph-dev — CHANGES_REQUESTED** [review]
What's the point of this?

File: src/app/controllers/OrderController.ts (hunk 1 above)
Comment (DenisMela) [reply src/app/controllers/OrderController.ts:441]: Where do we have fragments? What is this duplicating?

**Comment by DenisMela** [comment]
> What's the point of this?

The getOrderDetails query should return the order object, not just the _id of the object

**Review by ralph-dev — APPROVED** [review]
Lgtm but I think you have two back ticks next to each other

**Pull request merged**